README.md

```diff
@@ -9,7 +9,14 @@ The code is as open source as we can possibly make it (safely) and is powered by
 
 Head to [quartz.solar](https://quartz.solar) to find out more or to get in touch about using our Production services.
 
+### Running the apps in this project
 
+The following documentation explains how to get started with running each application:
+
+- [nowcasting-app](apps/nowcasting-app/README.md) - The main Quartz Solar UI, hosted at [app.quartz.solar](app.quartz.solar).
+- [nowcasting-website](apps/nowcasting-website/README.md) - The marketing website
```

Comment:
I would remove this, as its odd. Also the ux-demo as they are old. Sorry we should remove these folder out the github repo too.

Reply:
see #514

Reply:
Yep, thanks for this @mduffin95 – if you remove `nowcasting-website` and `ux-demos` now they've been removed from the repo, we can get this merged 👍